Paper.tex

```diff
@@ -443,13 +451,14 @@ \subsubsection{Block Header Validity}
 
 The nonce, $H_n$, must satisfy the relation:
 \begin{equation}
+\lable{eq:PoW}
 %\mathtt{PoW}(H, H_n) \dfrac{\leqslant}{den} \frac{2^{256}}{H_d}
 \mathtt{PoW}(H, H_n) \leqslant \frac{2^{256}}{H_d}
 \end{equation}
 
 Where $\mathtt{PoW}$ is the proof-of-work function (see section \ref{ch:pow}): this evaluates to an pseudo-random number cryptographically dependent on the parameters $H$ and $H_n$. Given an approximately uniform distribution in the range $[0, 2^{256})$, the expected time to find a solution is proportional to the difficulty, $H_d$.
 
-This is the foundation of the security of the blockchain and is the fundamental reason why a malicious node cannot propagate newly created blocks that would otherwise overwrite (``rewrite'') history. Because the nonce must satisfy this requirement, and because its satisfaction depends on the contents of the block and in turn its composed transactions, creating new, valid, blocks is difficult and, over time, requires approximately the total compute power of the trustworthy portion of the mining peers.
+Identity \eqref{eq:PoW} is the foundation of the security of the blockchain and is the fundamental reason why a malicious node cannot propagate newly created blocks that would otherwise overwrite (``rewrite'') history. Because the nonce must satisfy requirement \eqref{eq:PoW}, and because its satisfaction depends on the contents of the block and in turn its composed transactions, creating new, valid, blocks is difficult and, over time, requires approximately the total compute power of the trustworthy portion of the mining peers.
```

Comment:
This doesn't make sense - it's the PoW that we're talking about, not "Identity".